services/catalog/src/simcore_service_catalog/core/settings.py

```diff
@@ -62,11 +62,26 @@ class Config(_CommonConfig):
         env_prefix = "POSTGRES_"
 
 
+class ClientRequestSettings(BaseSettings):
```

Comment:
why not adding this to https://github.com/ITISFoundation/osparc-simcore/tree/master/packages/models-library/src/models_library/settings ?

Reply:
moved